lxd-migrate/go.mod

```diff
@@ -2,40 +2,39 @@ module github.com/canonical/lxd-pkg-snap/lxd-migrate
 
 go 1.18
```

Comment:
I guess that's something we should think about bumping to `1.20`.

Reply:
Yes please